cmd/runtimes/deepspeed/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM nvidia/cuda:12.4.1-devel-ubuntu22.04
```

Comment:
```suggestion
FROM mpioperator/base:v0.6.0 AS mpi
FROM nvidia/cuda:12.4.1-devel-ubuntu22.04
```

I want to avoid managing sshd config file in this repository.

Reply:
Good suggestion!